src/AWS.Deploy.CLI/AWSUtilities.cs

```diff
@@ -18,7 +15,7 @@ namespace AWS.Deploy.CLI
 {
     public interface IAWSUtilities
     {
-        Task<AWSCredentials> ResolveAWSCredentials(string? profileName, string? lastUsedProfileName = null);
```

Comment:
Removing the `lastUsedProfileName` as this method is never invoked with this argument.